pkg/pxc/service.go

```diff
@@ -281,6 +296,13 @@ func NewServiceHAProxy(cr *api.PerconaXtraDBCluster) *corev1.Service {
 	return obj
 }
 
+func NewServiceHaproxyName(cr *api.PerconaXtraDBCluster) types.NamespacedName {
```

Comment:
Conciser rename to HaproxyServiceNamespacedName

Reply:
fbb29694